fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/FilteredAggregationMaker.java

```diff
@@ -66,7 +66,7 @@ protected LogicalMetric makeInner(String metricName, List<String> dependentMetri
                 new TemplateDruidQuery(
                         ImmutableSet.of(filteredAggregation),
                         Collections.emptySet(),
-                        sourceMetric.getTemplateDruidQuery().getInnerQuery()
+                        sourceMetric.getTemplateDruidQuery().getInnerQuery().orElse(null)
```

Comment:
See [Andrew's comment](https://github.com/yahoo/fili/pull/485#discussion_r131694877)

Although I do agree it gets a little awkward here - it could probably use a comment on the constructor at least.